src/main/resources/lang/installer_es_ES.properties

```diff
@@ -0,0 +1,38 @@
+installer.title=Instalador de Fabric
+option.create.profile=Crear perfil
+option.show.snapshots=Mostrar snapshots
+progress.done=Hecho
+progress.done.start.server=Hecho, inicia el servidor ejectutando {0}
+progress.done.server=Servidor instalado correctamente
+progress.download.libraries=Descargando archivos necesarios
+progress.download.minecraft=Descargando el servidor de Minecraft
+progress.download.library.entry=Descargando librería {0}
+progress.exception.no.launcher.directory=¡No se encontró ningún directorio del lanzador!
+progress.generating.launch.jar=Generando JAR de lanzamiento del servidor
+progress.generating.launch.jar.library=Generando JAR de lanzamiento del servidor: {0}
+progress.installing=Instalando Fabric Loader {0} en el cliente
+progress.installing.server=Instalando Fabric Loader {0} en el servidor
+prompt.exception=Error
+prompt.exception.caused.by=Causado por
+prompt.exception.occurrence=¡Ocurrió un error!
+prompt.exception.suppressed=Suprimido
+prompt.install=Instalar
+prompt.loader.version=Versión de Loader:
+prompt.loading.versions=Cargando versiones
+prompt.launcher.open.body=Por favor, cierra el lanzador de Minecraft antes de la instalación para asegurar un funcionamiento correcto.\n\n¿Quieres continuar igualmente?
+prompt.launcher.open.tile=El lanzador de Minecraft está abierto
+prompt.game.version=Versión de Minecraft:
+prompt.ready.install=Listo para instalar
+prompt.select.location=Elige carpeta de instalación
+prompt.server.info.jar=El JAR del servidor de Minecraft oficial es necesario para ejecutar Fabric
+prompt.server.info.command=Usa este comando para iniciar el servidor
+prompt.server.info.scipt=O genera scripts de lanzamiento
+prompt.server.jar=Descargar JAR del servidor
+prompt.server.jar.valid=JAR del servidor de la {0} válido encontrado
+prompt.server.jar.invalid=No se encontró un JAR del servidor de la {0} válido
+prompt.server.generate=Generar
+prompt.server.overwrite=¿Estás seguro que quieres reemplazar los scripts de lanzamiento existentes?
```

Comment:
reemplazar seems weird, 'sobrescribir' might fit better here. it's generally what I see people use for overwrite, but it's just a small word change.

Side note, the English one says override instead of overwrite but whatever